project-mappings

```diff
@@ -100,9 +100,6 @@
 :beam-runners-flink-1.7-job-server-container :runners:flink:1.7:job-server-container
 :beam-runners-flink-1.7-job-server :runners:flink:1.7:job-server
 :beam-runners-flink-1.7 :runners:flink:1.7
-:beam-runners-flink-1.6-job-server-container :runners:flink:1.6:job-server-container
-:beam-runners-flink-1.6-job-server :runners:flink:1.6:job-server
-:beam-runners-flink-1.6 :runners:flink:1.6
```

Comment:
Looks good. Please also remove `:beam-runners-flink_2.11*` above.

Reply:
Yes, we can remove the flink-1.5 as well in this PR.

Reply:
I think the `` is mistake of the commit `https://github.com/apache/beam/pull/9632/files#diff-2bb618406f7ee4470a48343283f368a2R97`
So, I think we do not need remove it, and  it's better to correct the Artifact Id for flink-1.8.
What do you think?

Reply:
Makes sense to me.